src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -38,7 +38,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
   sys.addShutdownHook(shutdown())
 
   lazy val db = DatabaseUtil.db
-  val tezosNodeOperator = new TezosNodeOperator(new TezosNodeInterface(tezosConf, callsConf, streamingClientConf), batchingConf)
+  val tezosNodeOperator = new TezosNodeOperator(new TezosNodeInterface(tezosConf, callsConf, streamingClientConf), tezosConf.network, batchingConf)
```

Comment:
"for adding governance fields to the block table, this change is extraneous. It would have been better to open a ticket and do a separate pr, so we have more traceability into changes to conseil"

Reply:
We discussed this already, you're totally right, but it's not easy to make the right call when a big refactoring is in the working.

I will make a better effort of splitting smaller PRs in the future.
Still is not so easy to do that properly, as
 - it's not always clear during the development, how the final solution will look like, before trying to use it practically
 - doing a review on a "preparation refactor" is not easy if the changes have no clear application

That said, I will anyway make the effeot to improve our process,
thank you